timesketch/lib/utils.py

```diff
@@ -165,8 +165,18 @@ def read_and_validate_csv(
 
             # Normalize datetime to ISO 8601 format if it's not the case.
             try:
-                chunk['datetime'] = pandas.to_datetime(chunk['datetime'])
-
+                chunk['datetime'] = pandas.to_datetime(
```

Comment:
I opt to leave a comment what this will coerce will do to it.
```Lines with unrecognized datetime format will result in NaN (not available) as value and the event will be dropped in the next line after ```

Reply:
Will do.